beps/0001-notifications-system/README.md

```diff
@@ -160,89 +168,202 @@ The following backend service interfaces are added as part of this proposal.
 ```ts
 // TODO - We may want to add an additional wrapping here with interfaces for Notification and NotificationParameters
 
-interface SendNotificationRequest {
-  receivers: SignalReceivers;
+export type NotificationRecipients =
```

Comment:
missing list of links

Reply:
I think it should not have a list of links but one link per notification

Reply:
There are several reasons for having an option to have more than one link here.
https://github.com/backstage/backstage/pull/22510#discussion_r1475839682

Reply:
For me it sounds like an overkill, never seen a notification system in any system that has more than one link per notification. @Rugvip any thoughts on this?

Reply:
I am in favor of the `additionalLinks` and that being optional. I think it's not possible to have a notification without it taking you somewhere and thus the `link` should be mandatory. It's up to UI to decide how possible additional links are rendered.